tripleo-ciscoaci/deployment/neutron_opflex/neutron-opflex-agent-container-puppet.yaml

```diff
@@ -0,0 +1,202 @@
+    value:
+      service_name: neutron_opflex_agent
+      config_settings:
+        map_merge:
+          - get_attr: [NeutronBase, role_data, config_settings]
+          - get_attr: [RoleParametersValue, value]
+          - neutron::agents::ml2::ovs::local_ip: {get_param: [ServiceNetMap, NeutronTenantNetwork]}
+      puppet_config:
+        config_volume: neutron_opflex
+        puppet_tags:
+        step_config: |
+          include ::tripleo::profile::base::ciscoaci_neutron_opflex
+        config_image: {get_param: ContainerNeutronOpflexAgentImage}
+        # We need to mount /run for puppet_config step. This is because
+        # puppet-vswitch runs the commands "ovs-vsctl list open_vswitch ."
+        # when running vswitch::ovs::enable_hw_offload: true
+        # ovs-vsctl talks to the ovsdb-server (hosting conf.db)
+        # on the unix domain socket - /run/openvswitch/db.sock
+        volumes:
+          - /lib/modules:/lib/modules:ro
+          - /run/openvswitch:/run/openvswitch
+          - /etc/os-net-config:/etc/os-net-config:ro
+      kolla_config:
+        /var/lib/kolla/config_files/ciscoaci_neutron_opflex_agent.json:
+          command: /bin/supervisord -c /etc/neutron/neutron_opflex_supervisord.conf
+          config_files:
+            - source: "/var/lib/kolla/config_files/src/*"
+              dest: "/"
+              merge: true
+              preserve_properties: true
+          permissions:
+            - path: /var/log/neutron
+              owner: neutron:neutron
+              recurse: true
+      docker_config_scripts: {get_attr: [ContainersCommon, container_config_scripts]}
+      docker_config:
+        step_4:
+          ciscoaci_neutron_opflex_agent:
+            start_order: 14
+            image: {get_param: ContainerNeutronOpflexAgentImage}
+            net: host
+            pid: host
+            privileged: true
+            restart: always
+            depends_on:
+              - openvswitch.service
+            healthcheck:
+              test: /etc/neutron/neutron_opflex_healthcheck
+            ulimit: {get_param: DockerOpenvswitchUlimit}
+            volumes:
+              list_concat:
+                - {get_attr: [ContainersCommon, volumes]}
+                -
+                  - /var/lib/kolla/config_files/ciscoaci_neutron_opflex_agent.json:/var/lib/kolla/config_files/config.json:ro
+                  - /var/lib/config-data/puppet-generated/neutron_opflex/:/var/lib/kolla/config_files/src:ro
+                  - /var/lib/config-data/puppet-generated/neutron/etc/neutron/neutron.conf:/etc/neutron/neutron.conf:ro
+                  - /var/lib/config-data/puppet-generated/neutron/etc/neutron/metadata_agent.ini:/etc/neutron/metadata_agent.ini:ro
+                  - /lib/modules:/lib/modules:ro
+                  - /run/openvswitch/:/run/openvswitch/:shared,z
+                  - /run/netns:/run/netns:shared,z
+                  - /var/log/containers/neutron:/var/log/neutron
```

Comment:
We need the shared volume changes (used for sharing ep files and notification- socket) as well in both 
neutron-opflex-agent-container-puppet.yaml and opflex-agent-container-puppet.yaml files. 

Pls see here:
https://github.com/noironetworks/tripleo-ciscoaci/pull/43/files#diff-c23d17d0e76333d46a2ee1190520c4b9826846f16dd711d4956bdd126312242cR132-R133